### PR TITLE
[mr] Translations updated

### PR DESCRIPTION
**Pull request by krnlpawar**

### CODE REVIEW

**Comment by krnlpawar** [comment]
Hello, I've checked all Marathi [mr] translations and most of them were not grammatically correct, in this PR I've fixed them all.

File: locales/mr/validation.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 return [
     'accepted'             => ':attribute ला स्वीकार केला गेला पाहिजे.',
-    'accepted_if'          => 'The :attribute must be accepted when :other is :value.',
+    'accepted_if'          => ':other हे :value असेल तेव्हा हे फील्ड स्वीकारणे आवश्यक आहे.',
```
Comment (luisprmat) [inline locales/mr/validation.php:16]: Shouldn't the `:attribute` placeholder appear?

**Review by luisprmat — CHANGES_REQUESTED** [review]

File: locales/mr/validation.php (hunk 1 above)
Comment (krnlpawar) [reply locales/mr/validation.php]: fixed, please check.

**Review by luisprmat — APPROVED** [review]

**Comment by caouecs** [comment]
Thank you

**Pull request merged**